config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+  
+  resources :items, only: [:create, :destroy]
```

Comment:
I tried this, and I think it would take a lot of work to make it do the right thing. The most immediate problem is that now it interprets `users/sign_in` as if `sign_in` is the id of a user.

My original simple idea is that each user would only be able to see his or her own to-do items. In that case, there is no need to nest the items under user, since there would only be a single list of items relevant at any time (those of the current_user).